buildpatches/gazelle.patch

```diff
@@ -39,3 +39,79 @@ index f5ae5e5..74678b4 100644
      deps = ["@com_github_bazelbuild_buildtools//build"],
  )
  
+diff --git a/language/go/modules.go b/language/go/modules.go
```

Comment:
@fmeum this patch could be interesting to you.

Instead of using `go list` output as-is, I filtered it using `go mod download` output, which seems to accurately track the dependencies after [graph pruning](https://go.dev/ref/mod#graph-pruning). Essentially matching the content of `go.mod` file.

This massively prune down our deps.bzl file.